### ISSUE TITLE
Docker image not working on daemon mode

### ISSUE THREAD

**Issue by c-martinez** [silviodc/DockerCliopatria#1]
I just pulled the container from Dockerhub:

```$ docker pull silviodc/cliopatria```

And tried running on daemon mode:
```$ docker run --name "myCliopatria" -p 3020:3020 -p 3050:3050 -v $PWD/storage/:/opt/cliopatria/storage/ -d silviodc/cliopatria```

But when I browse to `http://localhost:3020/`, I cannot connect. I checked docker to see if it was exposing the right port:
```$ docker ps -a ```

But it turns out my container had already terminated:
```
CONTAINER ID        IMAGE                 COMMAND                 CREATED             STATUS                     PORTS               NAMES
0f12e87f4894        silviodc/cliopatria   "/bin/sh -c start.sh"   15 seconds ago      Exited (0) 2 seconds ago                       myCliopatria
```

Running it on interactive mode works fine:
```
$ docker run --name "myCliopatria" -p 3020:3020 -p 3050:3050 -it --rm silviodc/cliopatria
```

**Comment by silviodc** [comment]
Hi @c-martinez 

Thanks so much for your comments. 
I reproduced your error. Find here, what i figured out so far. 
_The instalation is going well in both ways (-it , -d). The problem is in the docker start_

Right now i'm busy with other projects and i will check the it more deep as soon as possible.

Therefore, if you need to use it at daemon mode, find here a solution that works:
1) leave the iterative mode
2) start the container again using `docker start myContainer` it will work at background

PS: It is not the most beaultiful solution, i will fix it in near future.

Best
Silvio

**Comment by c-martinez** [comment]
Hi @silviodc,

Your suggested solution works, but like you say, it is not the most beautiful. It would be nice to have a fix. I will also try to find a way to fix it, and if I manage, I will send a PR.

Best,
Carlos

**Comment by silviodc** [comment]
Hi @c-martinez 

I found a solution for the problem, find it hereafter:

Run the Cliopatria docker in background:
`docker run --name "myCliopatria" -p 3020:3020 -p 3050:3050 -v $PWD/storage/:/opt/cliopatria/storage/ -d -t cliopatria`

I just included the `-t` after `-d` in this command.
Now you will be able to run it in the background without leave and start the container.
PS: I updated the documentation including it.

Best,
Silvio
